CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 1.5.0
```

Comment:
uh, the changelog is outdated, it does not include the latest changes from 1.4.4, 1.5.0 and post-1.5.0.
next version will be 1.6.0

Reply:
oh, I committed my changed on top of master branch, but I just see now that there is also a develop branch, which is "behind" the master.  looks that something is wrong here.

Reply:
oh, I committed my changed on top of master branch, but I just see now that there is also a develop branch, which is "behind" the master.  looks that something is wrong here.